chalicelib_smaht/checks/audit_checks.py

```diff
@@ -84,3 +84,23 @@ def check_validation_errors(connection, **kwargs):
         check.summary = 'No validation errors'
         check.description = 'No validation errors found.'
     return check
+
+
+@check_function()
+def check_submitted_md5(connection):
+    """ Check that any submitted md5s are consistent with the ones we generated """
+    check = CheckResult(connection, 'check_submitted_md5')
+
+    search_url = 'search/?type=SubmittedFile&submitted_md5sum!=No+value&content_md5sum!=No+value&limit=100'
```

Comment:
Do we know that they are always submitting content md5s?

Reply:
I guess this will always get the most recent 100 files. If there are a lot of submissions and nobody looks at that check for a few days, we will miss potential warnings. Maybe a sliding date window would be better? Somebody still would need to look regularly.

Reply:
Hm.. if there aren't a lot of submissions, getting the last 100 is probably better. Then the check will keep failing is there is an issue

Reply:
Maybe the union of the last 100 files and the last N days would be best?

Reply:
They're not always submitting md5s but yeah it sounds like its rare for them to do it at all so am anticipating not seeing that many...

Reply:
Ok, in this case getting the last 500 is certainly sufficient.